example/Gruntconfig.json

```diff
@@ -5,8 +5,20 @@
   },
   "domain": "project.vm",
   "packages": {
-    "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
-    "projFiles": ["README*", "bin/**"]
+    "default": {
+      "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
+      "projFiles": ["README*", "bin/**"]
+    },
+    "acquia": {
+      "srcFiles": ["**", ".**", "!sites/default/**", "!sites/*/files/**", "!??/**", "!sites/all/modules/dev/**", "!sites/*/settings.php", "!sites/*/settings.db.inc", "!sites/*/settings.local.inc", "!.htaccess", "!.gitignore", "!.gitkeep"],
+      "sharedFiles": ["**", "!files/**", "!default.settings.php", "settings.php", "!settings.local.inc"],
+      "projFiles": ["README*", "bin/**"],
+      "root": "build/packages/acquia",
+      "paths": {
+        "docroot": "build/packages/acquia/docroot",
+        "shared": "build/packages/acquia/shared"
+      }
```

Comment:
Look at https://github.com/phase2/grunt-drupal-tasks/blob/master/CONFIG.md#package-settings to see the package.dest settings.

What is the role of the shared directory?